tests/.nftignore

```diff
@@ -17,8 +23,13 @@ GenomeBinning/QC/BUSCO/**/*.log
 GenomeBinning/QC/BUSCO/**/*.txt
 GenomeBinning/QC/BUSCO/**/short_summary*.{txt,json}
 GenomeBinning/QC/BUSCO/*/*{-busco.log,-busco.batch_summary.txt}
+GenomeBinning/QC/CheckM/*/{checkm.log,lineage.ms}
+GenomeBinning/QC/CheckM/*/bins/*/hmmer.{analyze,tree}.txt
+GenomeBinning/QC/CheckM/*/storage/*
 GenomeBinning/QC/CheckM2/**/DIAMOND_RESULTS.tsv
 GenomeBinning/QC/CheckM2/*/checkm2.log
+GenomeBinning/QC/QUAST/*/*.{pdf,html,log}
```

Comment:
To verify, there is no 'nice' 'completed successful' entry in the log file? Other two are fine